itools-packager/src/main/resources/itools.bat

```diff
@@ -48,17 +48,18 @@ if "%~1"=="" ( goto done ) else (
 :done
 
 set options=
-if "%powsybl_config_dirs%" == "" ( set powsybl_config_dirs=%installDir%\etc:%HOMEDRIVE%%HOMEPATH%\.itools)
+if "%powsybl_config_dirs%" == "" ( set powsybl_config_dirs=%installDir%\etc;%HOMEDRIVE%%HOMEPATH%\.itools)
```

Comment:
Replace set by setlocal for local variables

Reply:
`setlocal` is actually a command which allows all future sets to be only local and is used line 19.